sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -19,13 +19,13 @@
 namespace Azure.Storage.Test.Shared
 {
     [ClientTestFixture(
-        /* BlobClientOptions.ServiceVersion.V2019_02_02,
+        BlobClientOptions.ServiceVersion.V2019_02_02,
         BlobClientOptions.ServiceVersion.V2019_07_07,
         BlobClientOptions.ServiceVersion.V2019_12_12,
         BlobClientOptions.ServiceVersion.V2020_02_10,
         BlobClientOptions.ServiceVersion.V2020_04_08,
         BlobClientOptions.ServiceVersion.V2020_06_12,
-        BlobClientOptions.ServiceVersion.V2020_08_04, */ // https://github.com/Azure/azure-sdk-for-net/issues/20386
+        BlobClientOptions.ServiceVersion.V2020_08_04,
         StorageVersionExtensions.LatestVersion,
```

Comment:
Please keep it commented out in final version. We'll clean this up while working on "weekly pipeline" or when NUnit fix arrives.

Reply:
Nvm. I missed the 3rd point from description.